pkg/scanner/local/scan.go

```diff
@@ -222,7 +222,7 @@ func (s Scanner) misconfsToResults(misconfs []ftypes.Misconfiguration, options t
 
 // MisconfsToResults is exported for trivy-plugin-aqua purposes only
 func (s Scanner) MisconfsToResults(misconfs []ftypes.Misconfiguration) types.Results {
-	log.Logger.Infof("Detected config files: %d", len(misconfs))
+	log.Logger.Infof("Detected files with misconfig: %d", len(misconfs))
```

Comment:
I don't think it is correct to say that files with misconfig are detected, because the list of misconfigurations may only contain successfully passed checks.